libs/ml/benchmark/activations/activations.cpp

```diff
@@ -366,59 +367,97 @@ BENCHMARK_TEMPLATE(BM_RandomisedReluBackward, double, 1024)->Unit(benchmark::kMi
 BENCHMARK_TEMPLATE(BM_RandomisedReluBackward, double, 2048)->Unit(benchmark::kMicrosecond);
 BENCHMARK_TEMPLATE(BM_RandomisedReluBackward, double, 4096)->Unit(benchmark::kMicrosecond);
 
-template <class T, int N>
+template <class T>
 void BM_ReluForward(benchmark::State &state)
 {
   using TensorType = typename fetch::math::Tensor<T>;
-  TensorType input({1, N});
-  TensorType output({1, N});
+
+  fetch::ml::utilities::BM_Tensor_config<::benchmark::State> config{state};
+
+  TensorType input(config.shape);
+  TensorType output(config.shape);
+
+  input.FillUniformRandom();
 
   std::vector<std::shared_ptr<fetch::math::Tensor<T> const>> inputs;
   inputs.emplace_back(std::make_shared<TensorType>(input));
-  fetch::ml::ops::Relu<fetch::math::Tensor<T>> rm;
+  fetch::ml::ops::Relu<fetch::math::Tensor<T>> relu;
+
+  relu.SetBatchInputShapes({config.shape});
```

Comment:
You don't need to set batch input shapes now that ChargeForward takes the input shapes